doc/embedded/supported_boards.md

```diff
@@ -0,0 +1,126 @@
+| NORTHWOOD | [compiler set](https://toolchains.bootlin.com/releases_x86-64.html)       | [Sysroot and toolchain prefix](#NORTHWOOD)   |[Check this page](https://en.wikipedia.org/wiki/Pentium_4)              |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| COPPERMINE |  [compiler set](https://toolchains.bootlin.com/releases_x86-i686.html)   | [Sysroot and toolchain prefix](#COPPERMINE) |[Check this page](https://en.wikipedia.org/wiki/Pentium_III)             |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+
+
+### ARM11
+
+`-DTOOLCHAIN_PREFIX=/path/to/bootlin/toolchain/armv6-eabihf--glibc--stable-2024.05-1/bin/arm-buildroot-linux-gnueabihf- -DCMAKE_SYSROOT=/path/to/bootlin/toolchain/armv6-eabihf--glibc--stable-2024.05-1/arm-buildroot-linux-gnueabihf/sysroot`
+
+### CORTEXA7 
+
+`-DTOOLCHAIN_PREFIX=/path/to/bootlin/toolchain/armv7-eabihf--glibc--stable-2024.02-1/bin/arm-buildroot-linux-gnueabihf- -DCMAKE_SYSROOT=/path/to/bootlin/toolchain/armv7-eabihf--glibc--stable-2024.02-1/arm-buildroot-linux-gnueabihf/sysroot` 
+
+### CORTEXA8 
+
+`-DTOOLCHAIN_PREFIX=/path/to/bootlin/toolchain/armv7-eabihf--glibc--stable-2024.02-1/bin/arm-buildroot-linux-gnueabihf- -DCMAKE_SYSROOT=/path/to/bootlin/toolchain/armv7-eabihf--glibc--stable-2024.02-1/arm-buildroot-linux-gnueabihf/sysroot`
+
+### CORTEXA9  
+
+`-DTOOLCHAIN_PREFIX=/path/to/bootlin/toolchain/armv7-eabihf--glibc--stable-2024.02-1/bin/arm-buildroot-linux-gnueabihf- -DCMAKE_SYSROOT=/path/to/bootlin/toolchain/armv7-eabihf--glibc--stable-2024.02-1/arm-buildroot-linux-gnueabihf/sysroot`
+
+### CORTEXA15 
+
+`-DTOOLCHAIN_PREFIX=/path/to/bootlin/toolchain/armv7-eabihf--glibc--stable-2024.02-1/bin/arm-buildroot-linux-gnueabihf- -DCMAKE_SYSROOT=/path/to/bootlin/toolchain/armv7-eabihf--glibc--stable-2024.02-1/arm-buildroot-linux-gnueabihf/sysroot`
+
+### CORTEXA53 
+
+`-DTOOLCHAIN_PREFIX=/path/to/bootlin/toolchain/aarch64--glibc--stable-2024.02-1/bin/arm-buildroot-linux-gnueabihf- -DCMAKE_SYSROOT=/path/to/bootlin/toolchain/aarch64--glibc--stable-2024.02-1/arm-buildroot-linux-gnueabihf/sysroot`
+
+### CORTEXA72
+
+`-DTOOLCHAIN_PREFIX=/path/to/bootlin/toolchain/aarch64--glibc--stable-2024.02-1/bin/arm-buildroot-linux-gnueabihf- -DCMAKE_SYSROOT=/path/to/bootlin/toolchain/aarch64--glibc--stable-2024.02-1/arm-buildroot-linux-gnueabihf/sysroot`
+
+### CORTEXA76
+
+`-DTOOLCHAIN_PREFIX=/path/to/bootlin/toolchain/aarch64--glibc--stable-2024.02-1/bin/arm-buildroot-linux-gnueabihf- -DCMAKE_SYSROOT=/path/to/bootlin/toolchain/aarch64--glibc--stable-2024.02-1/arm-buildroot-linux-gnueabihf/sysroot`
```

Comment:
When I just ran this locally, I had to specify:

 * `-DTOOLCHAIN_PREFIX=/opt/bootlin/aarch64--glibc--stable-2024.02-1/bin/aarch64-buildroot-linux-gnu-`
 * `-DCMAKE_SYSROOT=/opt/bootlin/aarch64--glibc--stable-2024.02-1/aarch64-buildroot-linux-gnu/sysroot/`
 
 which is different than what you've got here---did I do something wrong?

Reply:
probably error from my side in copy paste from previous line,

Reply:
No worries, just wanted to make sure I hadn't missed something or was using the wrong buildroot or something :smile: